src/scripts/syncBreaches.js

```diff
@@ -11,7 +11,83 @@
 
 import { req, formatDataClassesArray } from '../utils/hibp.js'
 import { getAllBreaches, upsertBreaches } from '../db/tables/breaches.js'
+import { mkdir, readdir } from "node:fs/promises";
+import { resolve as pathResolve } from "node:path";
+import { finished } from 'stream/promises';
```

Comment:
nit: can we prefix this w/ [`node:stream/promises`](https://nodejs.org/dist/latest-v18.x/docs/api/stream.html#streams-promises-api) for consistency?
```suggestion
import { finished } from 'node:stream/promises';
```